### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by cgeroux**
This branch includes a number of small syntax and wording fixes to the earlier lessons

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -41,7 +41,8 @@ and to meet some of our community members.
 3.  If you are comfortable with Git,
     and would like to add or change material,
     you can submit a pull request (PR).
-    Instructions for doing this are [included below](#using-github).
+    For information about using pull requests see 
+    [github documentation on using pull requests](https://help.github.com/articles/using-pull-requests/).
```
Comment (gvwilson) [inline CONTRIBUTING.md]: We want to keep CONTRIBUTING.md the same across all repos, so please PR changes to it to the `styles` repo.

**Comment by gvwilson** [comment]
Thanks @cgeroux - can you please take CONTRIBUTING.md out of this PR and then I'll merge the rest?

**Comment by cgeroux** [comment]
Ok, will do a PR to the styles repo. Should I make a new PR for these changes not including the changes to CONTRIBUTING.md?

**Comment by cgeroux** [comment]
Actually looking at [https://github.com/swcarpentry/styles](url) I don't see a CONTRIBUTING.md file. Is that the repo you meant? Or am I missing something?

**Comment by gvwilson** [comment]
Yes please and thank you.

**Comment by gvwilson** [comment]
The text is embedded in bin/lesson_initialize.py - we don't want a 
CONTRIBUTING.md because per-site additions will result in endless merge 
conflicts, so we generate the file once when a new lesson is started.  
mumble mumble documentation mumble mumble...

**Comment by cgeroux** [comment]
Ok got it.. thanks. Will get the new pull requests out shortly. 

Is the "mumble mumble documentation mumble mumble" meaning, read the docs (if so where?) or do you mean, documentation is needed?

**Comment by gvwilson** [comment]
That's me reminding myself once again to write this stuff down - the 
pile of sticky notes telling me to do that is about a centimeter thick 
right now...

**Pull request closed without merging**